nncf/pruning/filter_pruning/algo.py

```diff
@@ -147,9 +148,69 @@ def _apply_binary_mask_to_module_weight_and_bias(module, mask, module_name=""):
                 bn_module = related_modules[PrunedModuleInfo.BN_MODULE_NAME]
                 _apply_binary_mask_to_module_weight_and_bias(bn_module, minfo.operand.binary_filter_pruning_mask)
 
+            if minfo.related_modules is not None and PrunedModuleInfo.DEPTHWISE_BN_NAME in minfo.related_modules \
+                    and related_modules[PrunedModuleInfo.DEPTHWISE_BN_NAME] is not None:
+                bn_module = related_modules[PrunedModuleInfo.DEPTHWISE_BN_NAME]
+                _apply_binary_mask_to_module_weight_and_bias(bn_module, minfo.operand.binary_filter_pruning_mask)
+
+    @staticmethod
+    def create_stats_table_for_pruning_export(old_modules_info, new_modules_info):
+        """
+        Creating a table with comparison of model params number before and after pruning.
+        :param old_modules_info: Information about pruned layers before actually prune layers.
+        :param new_modules_info: Information about pruned layers after actually prune layers.
+        """
+        table = Texttable()
+        header = ["Name", "Weight's shape", "New weight's shape", "Bias shape", "New bias shape",
+                  "Weight's params count", "New weight's params count",
+                  "Mask zero %", "Layer PR"]
+        data = [header]
+
+        for layer in old_modules_info.keys():
+            assert layer in new_modules_info
+
+            drow = {h: 0 for h in header}
+            drow["Name"] = layer
+            drow["Weight's shape"] = old_modules_info[layer]['w_shape']
+            drow["New weight's shape"] = new_modules_info[layer]['w_shape']
+            drow["Bias shape"] = old_modules_info[layer]['b_shape']
+            drow["New bias shape"] = new_modules_info[layer]['b_shape']
+
+            drow["Weight's params count"] = old_modules_info[layer]['params_count']
+            drow["New weight's params count"] = new_modules_info[layer]['params_count']
+
+            drow["Mask zero %"] = old_modules_info[layer]['mask_pr']
+
+            drow["Layer PR"] = 1 - new_modules_info[layer]['params_count'] / old_modules_info[layer]['params_count']
+            row = [drow[h] for h in header]
+            data.append(row)
+        table.add_rows(data)
+        return table
+
+    # pylint: disable=protected-access
     def export_model(self, filename, *args, **kwargs):
         """
         This function saving model without actually pruning the layers, just nullifies the necessary filters by mask.
```

Comment:
@mkaglins The description seems out of date. The model is actually pruned with this diff.